src/app/modules/ix-table/components/ix-table-body/cells/ix-cell-state-button/ix-cell-state-button.component.ts

```diff
@@ -41,6 +41,19 @@ export class IxCellStateButtonComponent<T> extends ColumnComponent<T> implements
   dialogService: DialogService = inject(DialogService);
   errorHandler: ErrorHandlerService = inject(ErrorHandlerService);
 
+  private readonly rowUpdateEffect = effect(() => {
+    const row = this.row();
+    const job = !row || !this.getJob ? undefined : this.getJob(row);
+
+    if (!job) {
+      return;
+    }
+
+    this.state.set(job.state);
+  }, {
+    allowSignalWrites: true,
+  });
```

Comment:
We can keep it for now, but in the future we should try to refactor code to avoid having effects when possible and rely on `computed` signals more.